### DISCUSSION TITLE
CSV to CDT

### DISCUSSION THREAD

**Discussion by laiajulianamu** [dankelley/oce#1787] [Q&A] [answered]
Hi. I have a data set with temperature, density, salinity, pressure and depth. I have my data in a cvs sheet and I would like to know how can I convert all my set into a CTD object for analyzing

**Comment by richardsc** [comment, marked as answer]
Hi @laiajulianamu,

I recommend you read the documentation for the `as.ctd()` function, e.g.

https://dankelley.github.io/oce/reference/as.ctd.html

which should accomplish what you need. Depending on how many fields (data and metadata) you need to add, you might also want to look at the `oceSetMetadata()` and `oceSetData()` functions for adding additional fields.

Cheers,
Clark

**Reply by laiajulianamu** [reply]
Hi Clark. Thank you very much for your response. I checked the  funtion but I can´t add all data at the sametime.
